### ISSUE TITLE
Segmentation fault in keras.backend.temporal_padding

### ISSUE THREAD

**Issue by mjkim720** [tensorflow/tensorflow#40906]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): v2.2.0-rc4-8-g2b96f3662b 2.2.0
- Python version: 3.7.6
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**
Segfault occurs when passing the tuple of large values for `padding`.

**Describe the expected behavior**
No segfault.

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.
```python
import tensorflow as tf

x = [[[3, 2, 3],
        [1, 3, 4]],

       [[3, 1, 2],
        [3, 2, 4]],

       [[4, 4, 2],
        [1, 1, 1]]]
padding = (1130323445, 1510667856)

tf.keras.backend.temporal_padding(x, padding)
```

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.
```
Segmentation fault (core dumped)
```

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with TF v2.2 and TF-nightly. Please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/234812d20aac48606c491b0f6bd48d1b/40906.ipynb). Thanks!

**Comment by saikumarchalla** [comment]
Was able to reproduce the issue  using TF 2.5 and colab crashes as expected. Please find the gist [here](https://colab.research.google.com/gist/saikumarchalla/9dacdfb9d212caca73dd654aa2ded7c3/untitled85.ipynb).Thanks!

**Comment by jvishnuvardhan** [comment]
@mjkim720 It is working as expected when the `padding` is within a range. However, when passing large padding as in your example, padding = (1130323445, 1510667856), segfault occurs as it couldn't allocate such a high memory. 

Please note that `temporal_padding` pads zeros at the start and end of dim 1. 

For example, if the input tensor is of shape (3, 2, 3), passing a padding of (2,2) results in a tensor of shape (3, 6, 3). In your case, the resulting tensor is of shape (3,2640991303,3) which requires huge memory. Thanks!
